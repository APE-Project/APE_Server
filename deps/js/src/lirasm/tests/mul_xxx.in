; 46340 * 46340 < 2^31, and will not overflow.
big = immi 46340

res = muli big big

reti res
