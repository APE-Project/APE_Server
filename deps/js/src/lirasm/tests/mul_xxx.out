Output is: 2147395600
